.yarn/versions/aa7247ea.yml

```diff
@@ -0,0 +1,16 @@
+releases:
+  "@radix-ui/react-alert-dialog": patch
+  "@radix-ui/react-context-menu": patch
+  "@radix-ui/react-dialog": patch
+  "@radix-ui/react-dismissable-layer": patch
+  "@radix-ui/react-dropdown-menu": patch
+  "@radix-ui/react-menu": patch
+  "@radix-ui/react-navigation-menu": patch
+  "@radix-ui/react-popover": patch
+  "@radix-ui/react-select": patch
+  "@radix-ui/react-toast": patch
+  "@radix-ui/react-tooltip": patch
+
+declined:
+  - primitives
+  - "@radix-ui/react-hover-card"
```

Comment:
Looks like a slip-up here, but `HoverCard` should be part of the version change too as it depends on `DismissableLayer`.

Reply:
I didn't bump hover-card because it's not possible to open a hover card with the keyboard, but I suppose it is possible to close them. I'll update that now.

Reply:
Thanks! Yeah it has the same close on escape behaviour.